blaze-server/src/test/scala/org/http4s/server/blaze/BlazeServerMtlsSpec.scala

```diff
@@ -12,6 +12,7 @@ import org.http4s.{Http4sSpec, HttpApp}
 import scala.concurrent.duration._
 import scala.io.Source
 import scala.util.Try
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
If it's expected to use `global` explicitly only, it's better to
```scala
import scala.concurrent.ExecutionContext.global
```
although I'd recommend to
```scala
import scala.concurrent.ExecutionContext
```
and refer to `global` in a more clear way: `BlazeServerBuilder[IO](ExecutionContext.global)`

Reply:
As I can see, some code below imports the implicit `global`, some other – the explicit one. But I cannot figure out the difference. Is seems to me that `global` is referenced explicitly everywhere, isn't it?

Reply:
That's true.  They point to the same thing, and we probably imported from `Implicits` when this ticket originally set out to make it implicit.  But it would be marginally better if it just came from `ExecutionContext.global`.  I can mop that up when I backport it to the right branch, unless @albertoadami beats me to it.

Reply:
@rossabaker  I can do the changes directly to this PR here, it's a 5 minutes work.
 I think that's better to use the `scala.concurrent.ExecutionContext.global`.

Reply:
I changed the global import inside the tests, but now I get a test error on **Scala 2.12.11, Java adopt@1.8**, but shouldn't be due to my commit.